### DISCUSSION TITLE
Generación de números aleatorios en el TP 6

### DISCUSSION THREAD

**Discussion by IoachimusRoderici** [INGCOM-UNRN-P1/cursada-2023#187] [General de la cátedra]
La consigna del TP 6 sugiere usar la expresión 

```C
rand() % max + min
```

para generar números aleatorios entre min y max. Esta expresión no genera números en el rango correcto. La correcta sería

```C
rand() % (max-min) + min
```

para el intervalo $[min,\ max)$.

**Comment by martinvilu** [comment]
Excelente aporte del compañero.

**Comment by JeanFranco99** [comment]
![IMG-20230929-WA0028.jpg](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/83616156/b412d55d-ed3b-4b45-a668-ff85be9e3490)

Hago un mini aporte 
Ahí está mejor explicado el por qué se hace así
